cmd/jimmsrv/service/service.go

```diff
@@ -453,6 +453,7 @@ func NewService(ctx context.Context, p Params) (*Service, error) {
 	}
 
 	s.mux.Handle("/api", jujuapi.APIHandler(ctx, &s.jimm, params))
+	RegisterModelHTTPEndpoints(s.mux, &s.jimm)
```

Comment:
Shouldn't we make methods for:

- RegisterModelWSEndpoint
- RegisterControllerWSEndpoint
- RegisterReBACAdminEndpoints
- RegisterDebugEndpoint
- RegisterMetricsEndpoint

But still, I think having the group top level and explaining what middlewares it has so we can move routes more easily around depending on the middleware they need is better. I.e., all need tracing, and logging, so a top level group with those middle wares would embed other groups. Let's revisit this together